README.md

```diff
@@ -1,2 +1,79 @@
-# jenkinsfile-runner-test-framework
-Test framework for Jenkinsfile Runner
+# Jenkinsfile Runner Test Framework
+This repository provides a test harness for verifying Jenkinsfile Runner Docker images. That image can be already built or it can be built using Custom War Packager and the set of functions provided by the framework.
```

Comment:
set of utility functions maybe? Just to empower the idea that the framework is just using another product to build the image.